scripts/bash/enceladus_env.template.sh

```diff
@@ -53,13 +55,14 @@ CONF_DEFAULT_DRIVER_MEMORY=""
 CONF_DEFAULT_DRIVER_CORES=""
 CONF_DEFAULT_EXECUTOR_MEMORY=""
 CONF_DEFAULT_EXECUTOR_CORES=""
-CONF_DEFAULT_EXECUTOR_CORES=""
+CONF_DEFAULT_DRA_EXECUTOR_MEMORY=""
+CONF_DEFAULT_DRA_EXECUTOR_CORES=""
```

Comment:
Same as above

Reply:
This can be done on the Environment Level